features/machine/machine_healthcheck.feature

```diff
@@ -166,5 +166,44 @@ Feature: MachineHealthCheck Test Scenarios
      | Ensuring a requeue happens in 15m0  |
     """
    Then the machine should be remediated
+ 
+  # @author miyadav@redhat.com
+  # @case_id OCP-29857
+  @admin
+  @destructive
```

Comment:
Since we are creating a malformed mhc, we don't really have to clone a machineset. We can create the mhc against a current machineset and check the mhc log directly.  the `@destructive` tag can be removed